src/components/v2/Layout/Header/index.tsx

```diff
@@ -29,17 +32,13 @@ const Header = () => {
     <AppBar position="relative" css={styles.appBar}>
       <Toolbar css={styles.toolbar}>
         {title && <h3>{title}</h3>}
-        <Box
-          flexDirection="row"
-          display="flex"
-          flex={1}
-          justifyContent="right"
-          css={styles.rightItemContainer}
-        >
-          <XvsCoinInfo css={styles.rightItemPaper} className="coinInfo" />
-          <VaiCoinInfo css={styles.rightItemPaper} className="coinInfo" />
-          <ConnectButton css={styles.rightItemPaper} title={t('header.connectButton.title')} />
-        </Box>
+
+        {!isOnMobile && (
```

Comment:
Any reason not to use 'display: none;' here? It is nicer to let css handle this then listening for window changes in JS.